.clang-format

```diff
@@ -0,0 +1,14 @@
+---
+Language: Cpp
+BasedOnStyle: Google
+
+ColumnLimit: 120
+NamespaceIndentation: Inner
+
+# Scaled by a factor of 2 such that the base indent is 4
+AccessModifierOffset: -2
+ConstructorInitializerIndentWidth: 8
+ContinuationIndentWidth: 8
```

Comment:
Why deviate from Google's style for these?

Reply:
I think it's "Scaled by a factor of 2 such that the base indent is 4", not sure why that's important though.

Reply:
@boothby , do you have insight? I know you use the same format [in minorminer](https://github.com/dwavesystems/minorminer/blob/main/.clang-format).

Reply:
If there is no specific reason, I would probably prefer to default to Google's style.

Reply:
Huh.  I'm not actually sure.  Brad set that up and I've never changed it.